src/components/application_manager/include/application_manager/app_launch/app_launch_data_json.h

```diff
@@ -53,7 +52,11 @@ class AppLaunchDataJson : public AppLaunchDataImpl {
    * @brief Constructor of AppLaunchDataJson object
    */
   AppLaunchDataJson(const AppLaunchSettings& settings,
-                    resumption::LastState& last_state);
+                    resumption::LastStateWrapperPtr last_state_wrapper);
+
+  DEPRECATED AppLaunchDataJson(const AppLaunchSettings& settings,
+                               resumption::LastState&) = delete;
```

Comment:
@YarikMamykin do we really need to ` = delete` here?

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/5123861995b84627338520ff72ca611a3689295a